go.mod

```diff
@@ -55,6 +55,7 @@ require (
 	github.com/posener/complete v1.2.3
 	github.com/spf13/cobra v1.1.1 // indirect
 	github.com/stretchr/testify v1.7.0
+	github.com/vancluever/dawdle v0.3.0
```

Comment:
This does need to move into the HC org prior to merging.

Reply:
Done!